examples/src/main/scala/org/apache/spark/examples/ml/LogisticRegressionSummaryExample.scala

```diff
@@ -30,11 +30,11 @@ object LogisticRegressionSummaryExample {
   def main(args: Array[String]): Unit = {
     val conf = new SparkConf().setAppName("LogisticRegressionSummaryExample")
     val sc = new SparkContext(conf)
-    val sqlCtx = new SQLContext(sc)
-    import sqlCtx.implicits._
+    val sqlContext = new SQLContext(sc)
+    import sqlContext.implicits._
```

Comment:
Is this import still needed? (I forget.) It's weird to use a variable name in the import isn't it? (I know it was already like that.) Otherwise this does look like all instances of `sqlCtx` in examples, yes.